jetty-home/src/main/resources/modules/setuid.mod

```diff
@@ -9,7 +9,7 @@ changing to a restricted user (e.g. Jetty).
 server
 
 [lib]
-lib/setuid/jetty-setuid-java-1.0.4.jar
+lib/setuid/jetty-setuid-jna-2.0.0.jar
```

Comment:
Could you please confirm that adding a `lib/setuid/jna-5.13.0.jar` line isn't needed?